### DISCUSSION TITLE
majority report

### DISCUSSION THREAD

**Discussion by hildeb** [oetiker/SmokePing#280] [Q&A]
I'm running a smokeping setup with one master and three slaves
Anyway, some systems are being pinged from all three slave
systems, and the master system should alert only if at least two of
the three secondaries report an error condition (basically a majority
report).

Can that be done somehow? (smokeping 2.7.x)

**Comment by moetiker** [comment]
No that's not possible, alerts are only matching one graph... if you are interested that this functionality are added to SmokePing please contact @oetiker .
